model/group.go

```diff
@@ -43,6 +43,14 @@ type Group struct {
 	MemberCount  *int        `db:"-" json:"member_count,omitempty"`
 }
 
+type GroupNonAdmin struct {
```

Comment:
not sure why we need to create another struct for this, can we not just sanitize the values when we get groups back? i.e. sanitize them for non-admins. We can include the `omitempty` tag for fields we want to remove. Thoughts?

Reply:
Spoke to @hahmadia, regarding why we can't use `omitempty` as for the regular case we want to include fields with default values.

Suggesting to use [anonymous struct](https://www.geeksforgeeks.org/anonymous-structure-and-field-in-golang/) for marshalling purposes since it'll _only_ be used for marshalling, better t o not create a special struct that's a first class entity as opposed to a anonymous struct only used for marshalling.

Reply:
@ali-farooq0 That's a great point and it totally makes sense why it would be better to use anonymous struct rather than creating a new struct in our model file. I have made the changes to reflect your suggestion. Thanks! 😄